src/MerQure.Tools/Exceptions/MerqureToolsException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace MerQure.Tools.Exceptions
+{
+    public class MerqureToolsException : Exception
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this implementation of 'ISerializable' to conform to the recommended serialization pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3925)